cpp/src/arrow/engine/substrait/serde_test.cc

```diff
@@ -42,9 +55,35 @@ using testing::UnorderedElementsAre;
 namespace arrow {
 
 using internal::checked_cast;
-
+using internal::hash_combine;
 namespace engine {
 
+Status WriteParquetData(const std::string& path,
+                        const std::shared_ptr<fs::FileSystem> file_system,
+                        const std::shared_ptr<Table> input) {
+  EXPECT_OK_AND_ASSIGN(auto buffer_writer, file_system->OpenOutputStream(path));
+  PARQUET_THROW_NOT_OK(parquet::arrow::WriteTable(*input, arrow::default_memory_pool(),
+                                                  buffer_writer, /*chunk_size*/ 1));
+  return buffer_writer->Close();
+}
```

Comment:
Substrait supports IPC now.  Can we write this test with IPC?

Reply:
Let me take a look.

Reply:
You mean `ORC`?

Reply:
In Substrait the IPC format is called `arrow` with `ArrowReadOptions`.